palace/cmake/PkgConfigHelpers.cmake

```diff
@@ -51,19 +51,20 @@ int main()
   return 0;
 }
 ")
+  message(STATUS "USING _libceed_target == ${_libceed_target}")
   try_compile(
     LIBCEED_TEST_COMPILED
     ${LIBCEED_LIB_TEST_DIR}
     ${LIBCEED_LIB_TEST_CPP}
     LINK_LIBRARIES ${_libceed_target}
     OUTPUT_VARIABLE LIBCEED_TEST_COMPILE_OUTPUT
   )
-  # message(STATUS "LIBCEED_TEST_COMPILE_OUTPUT: ${LIBCEED_TEST_COMPILE_OUTPUT}")
   if(LIBCEED_TEST_COMPILED)
-    # message(STATUS "libCEED test program - Successful")
+    message(STATUS "libCEED test program - Successful")
     set(${_libceed_test_success} TRUE PARENT_SCOPE)
   else()
-    # message(STATUS "libCEED test program - Failed")
+    message(STATUS "libCEED test program - Failed")
+    message(STATUS "LIBCEED_TEST_COMPILE_OUTPUT: ${LIBCEED_TEST_COMPILE_OUTPUT}")
```

Comment:
I don't see any harm in leaving this in...

That being said, do we want to keep using this test? I find it sometimes fails, and sometimes passes, and often isn't really indicative of if libCEED is working at all.